src/server/pegasus_server_impl.cpp

```diff
@@ -2438,6 +2442,25 @@ void pegasus_server_impl::update_rocksdb_iteration_threshold(
     }
 }
 
+void pegasus_server_impl::update_validate_partition_hash(
```

Comment:
```suggestion
void pegasus_server_impl::update_check_partition_hash(
```

Be consistent with the concept "check_partition_hash", it would be ambiguous.